assets/js/workflows/usb.js

```diff
@@ -208,15 +208,17 @@ class USBWorkflow extends Workflow {
         this.onConnected();
     }
 
-    _hostFolderChanged() {
+    async _hostFolderChanged() {
```

Comment:
If you're making this asynchronous, please add an await before each call to this like on line #202.

Reply:
Done.